src/Data/Connection/CommentConnectionResolver.php

```diff
@@ -48,6 +48,7 @@ protected function prepare_query_args( array $args ): array {
 		 */
 		$query_args['orderby'] = 'comment_date';
 
+		
```

Comment:
Flagging the extra new-lines here, but I'm gonna `update phpcs.xml.dist` so these get fixed automatically by `phpcbf`